src/FSharp.Stats/List.fs

```diff
@@ -189,4 +189,20 @@ module List =
     let inline covBy f (list: 'T list) : 'U =
         list
         |> List.map f
-        |> covOfPairs
\ No newline at end of file
+        |> covOfPairs
+
+type List() =
+
+    /// Creates an list from an given interval.
+    /// start: start value (is included)
+    /// stop: end value (by default is included )
+    /// Num: sets the number of elements in the list. If not set, stepsize = 1.
+    /// IncludeEndpoint (default = true): If false, list does not contain stop value
+    static member linspace(start:float,stop:float,?Num:int,?IncludeEndpoint:bool) : float list = 
+        
+        let includeEndpoint = defaultArg IncludeEndpoint true
+
+        if Num.IsSome then 
+            Seq.linspace(start,stop,Num.Value,includeEndpoint) |> List.ofSeq
```

Comment:
I might be naive here and don't know enough about lazy evaluation of sequences, but are we creating a sequence and then iterating it to create a list from it, meaning this results in double the runtime? At least for array i can imagine that there could be a more optimized way of initializing it. However if small, the speed decrease might be worth it to have no code duplication.

Reply:
I started implementing it in the `Array` module because I assume it will be used here most often. After consideration of list, vector and sequences I moved it to `Seq` and refererenced it accordingly. It would definetely cause performance loss but maintaing four copies of the same method would be cumbersome. I'm going to keep it in mind and if efficiancy problems arise specialized functions will be added.